README.md

```diff
@@ -10,13 +10,12 @@ This plugin  is based on [Embedded PostgreSQL Server library](https://github.com
 
 ### pgServerVersion
 
-The plugin currently supports next PostgreSQL versions: 9.4.10, 9.5.5, 9.6.1
+The plugin currently supports next PostgreSQL versions: 9.5.7, 9.6.5, 10.0
 
 You also may use aliases:
-* 9.4 -> 9.4.10
```

Comment:
It would be great to keep 9.4 because lot of my customers use this version on PROD and it should be very strong reason why they should upgrade PgSQL to higher version...

Reply:
Right. I somehow was under the impression that Postgres only supported the current and 2 older versions, but it's clear [from this page](https://www.postgresql.org/support/versioning/) 9.4 is still supported for some time to come.

I'll see if I can add 9.4 back in without changes to `postgres-embedded`.